cmd/fossa/analyze.go

```diff
@@ -71,25 +70,25 @@ func doAnalyze(modules []moduleConfig, allowUnresolved bool) (analysis, error) {
 	for _, moduleConfig := range modules {
 		builder, m, err := resolveModuleConfig(moduleConfig)
 		if err != nil {
-			return nil, errors.New("failed to resolve modules: " + err.Error())
+			return nil, fmt.Errorf("failed to resolve modules: " + err.Error())
```

Comment:
The point of using `fmt.Errorf` is to use the format string. Here, this would be `fmt.Errorf("failed to resolve modules: %s", err.Error())`. Likewise below.